wp-includes/service-workers.php

```diff
@@ -236,4 +236,14 @@ function wp_default_service_workers( $service_workers ) {
 				);
 		}
 	}
+
+	add_action( 'wp_admin_service_worker', array( $service_workers, 'precache_admin_assets' ), 9 );
```

Comment:
Not sure if that's the best place where to add precaching for admin assets. Thoughts, @westonruter?

Reply:
@felixarntz With the change to `WP_Service_Worker_Scripts_Integration` and `WP_Service_Worker_Styles_Integration` to add them to both the front and admin scopes, then the integration will run at `wp_admin_service_worker` priority 10:

https://github.com/xwp/pwa-wp/blob/57ebfa7652cd6e485056c7d11417acb726a7e87d/wp-includes/service-workers.php#L219-L221

This means that in order to set the `precache` flags on admin-specific scripts and styles that the flag has to be set at priority 9 of the`wp_admin_service_worker` action. Something is missing here.